views/package_format_json.dt

```diff
@@ -398,6 +398,9 @@ block body
 					p <code>"default": true</code> - Choose an optional dependency by default.
 					p With this set to <code>true</code>, the dependency will be chosen by default if no dub.selections.json exists yet. If omitted, this attribute defaults to <code>false</code>. Note that this only has an effect if the "optional" attribute is set to <code>true</code>.
 					p Note that versions prior to 0.9.25 will ignore this attribute.
+				li
+					p Since v1.24.0, complex variant can also include <a href="#build-settings">build settings</a> fields to be applied to dependencies.
```

Comment:
```suggestion
					p Since v1.24.0, <a href="#build-settings">build settings</a> fields can be used to be applied to dependencies.
```
I think this makes grammatically more sense with the description of this list (`The following attributes can be used to to control how a dependency is resolved`) as well as not mentioning again that this is the complex variant.

ditto for SDL format

Reply:
> not mentioning again that this is the complex variant.

makes sense